app/src/test/java/org/mozilla/fenix/nimbus/NimbusBranchesStoreTest.kt

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.nimbus
+
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.mozilla.experiments.nimbus.Branch
+
+class NimbusBranchesStoreTest {
+
+    private lateinit var nimbusBranchesState: NimbusBranchesState
+    private lateinit var nimbusBranchesStore: NimbusBranchesStore
+
+    @Before
+    fun setup() {
+        nimbusBranchesState = NimbusBranchesState(branches = emptyList())
+        nimbusBranchesStore = NimbusBranchesStore(nimbusBranchesState)
+    }
+
+    @Test
+    fun testUpdateBranches() = runBlocking {
```

Comment:
Nit: Should be we follow the same name convention as other tests using `` [backticks](https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods)?